src/common/common/clipboard.h

```diff
@@ -83,10 +83,17 @@ void guac_common_clipboard_free(guac_common_clipboard* clipboard);
 /**
  * Informs users that clipboard is operational.
 
- * @param client The client to send the clipboard contents on.
+ * @param client The client to which clipboard became operational.
  */
 void guac_common_clipboard_operational_send(guac_client* client);
 
+/**
+ * Informs owner that that the remote clipboard was updted.
```

Comment:
```suggestion
 * Informs owner that that the remote clipboard was updated.
```

Reply:
@SpeeddY There's also two 'that's in that line....